.github/workflows/presubmit.yml

```diff
@@ -78,17 +89,32 @@ jobs:
             -DCMAKE_CXX_COMPILER="$(which clang++)" \
             -DCMAKE_C_COMPILER="$(which clang)" \
             -DCLKP_BUILD_TESTS=1 \
-            -DCMAKE_BUILD_TYPE=Release
+            -DCMAKE_BUILD_TYPE=Release \
+            -DSPIRV_DISASSEMBLY=ON \
+            -DCMAKE_PREFIX_PATH="$(pwd)/third_party/spirv-tools/install"
           cmake --build build_release
       - name: Tests
         run: |
           mkdir -p $(pwd)/kernels
+          echo "Testing with OpenCL C source (.cl file):"
```

Comment:
I think we should be able to have only one run, with both `gpu.cl` and `gpu.spv` as arguments.